tests/node_api.rs

```diff
@@ -2,232 +2,206 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // These are E2E test samples, so they are ignored by default.
-
-use std::str::FromStr;
-
-use bee_message::{output::OutputId, payload::transaction::TransactionId, MessageId};
+use bee_message::{
+    output::OutputId,
+    payload::{transaction::TransactionId, Payload},
+    MessageId,
+};
 use iota_client::{
+    bech32_to_hex,
     node_api::indexer::query_parameters::QueryParameter,
+    request_funds_from_faucet,
     secret::{mnemonic::MnemonicSecretManager, SecretManager},
+    Client,
 };
 
-const DEFAULT_NODE_URL: &str = "http://localhost:14265";
+const DEFAULT_DEVNET_NODE_URL: &str = "http://localhost:14265";
+const DEFAULT_DEVNET_FAUCET_URL: &str = "http://localhost:14265";
```

Comment:
Maybe also add a default const for the bech32 HRP and use that for address generation and also use it then for `rms1qzt0nhsf38nh6rs4p6zs5knqp6psgha9wsv74uajqgjmwc75ugupx4aaacx` (needs to be regenerated with the correct hrp)